source/_integrations/dnsip.markdown

```diff
@@ -4,72 +4,18 @@ description: Instructions on how to integrate a DNS IP sensor into Home Assistan
 ha_category:
   - Network
 ha_iot_class: Cloud Polling
+ha_config_flow: true
 ha_release: '0.40'
 ha_domain: dnsip
 ha_platforms:
   - sensor
 ---
 
-The `dnsip` sensor will expose an IP address, fetched via DNS resolution, as its value. There are two operational modes:
+The `dnsip` sensor will expose an IP address, fetched via DNS resolution, as its value. It provides both IPv4 and IPv6 lookup as separate sensors depending on accessability to resolvers.
 
-1. When you enable the sensor with minimal configuration, it will query the [OpenDNS](https://www.opendns.com/) nameservers with the hostname `myip.opendns.com`, which will resolve to your external/public IP address.
+1. When you enable the sensor with default value, it will query the [OpenDNS](https://www.opendns.com/) nameservers with the hostname `myip.opendns.com`, which will resolve to your external/public IP address.
```

Comment:
```suggestion
1. When you enable the sensor with the default value, it will query the [OpenDNS](https://www.opendns.com/) nameservers with the hostname `myip.opendns.com`, which will resolve to your external/public IP address.
```